requirements.txt

```diff
@@ -0,0 +1,8 @@
+abjad
+numpy
+tqdm
+nltk
+pillow
+pytorch-lightning
+torch
+transformers
```

Comment:
versions?

Reply:
I was only able to train this with one machine running Arch and one running Ubuntu 22.04. As we are targeting ROS1 with Ubuntu 20.04 the python major version differs and pinning the versions to my version might be risky and lead to some packages not working or unnecessary compilation from source by pip. Overall pining versions is important, but I would be careful doing it blindly.

Reply:
alright let's do that in a future commit